From 56435da8d33b5245d16795355b453bca952d672f Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 25 Jan 2023 19:20:51 +0000
Subject: [PATCH 08/10] MANTIS 0063780 extension: BER: Erweiterung Detaildaten
 auf PLF Daten Seite platform.idl add DeviceType.BSS add DataType.BSM

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1762 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 4dcd1862..3730b645 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "5.10";
+    const string interface_version = "5.11";
 
     /** Platform interface for the device components. Via this interface the device
         components communicate with the platform manager internally. */
@@ -259,14 +259,16 @@ module PlatformManagement
       GPP,
       ATB,
       BPP,
-      BTP
+      BTP,
+      BSS
     };
 
     enum DataType
     {
       AEA,
       SVG,
-      PDF
+      PDF,
+      BSM
     };
 
     struct LoggingCommandData
-- 
2.41.0.windows.1

